scss/functions/_utilities-map.scss

```diff
@@ -1,17 +1,19 @@
 // Builds a map of utility classes.
 @function build-utilities-map($grid-breakpoints: $grid-breakpoints, $utilities: $utilities) {
-
   // Build a breakpoint map that does not include the zero breakpoint.
   $breakpoints-map: ();
   @each $name, $min in $grid-breakpoints {
     @if $min != 0 {
-      $breakpoints-map: map-merge($breakpoints-map, (
-        "-" + $name: $name,
-      ));
+      $breakpoints-map: map-merge(
+        $breakpoints-map,
+        (
+          "-" + $name: $name,
+        )
+      );
     }
   }
 
-  $utilities-map: ();
+  $utilities-map: () !default;
```

Comment:
Is adding `!default` also part of the "linter" operation?
I usually try to have an isolated commit where I _only_ do style and whitespace changes, that do not change functionality.
This change does change functionality - in a good way, I would think. So could be in separate commit.